.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestBase.cs

```diff
@@ -334,7 +255,10 @@ public void TestIpAddress()
             {
                 Debugger.Break();
             }
+        }
 
+        private void GeneralMainTest(bool testUnit = false, bool testType = false)
```

Comment:
Rename the method to ValidateResults

Reply:
The parameter could be a list of extra keys to check, instead of multiple booleans. In the future as we have more keys, we don't need extra parameters.
By default "value" is always checked.